packages/eslint-config-react-app/index.js

```diff
@@ -189,14 +189,7 @@ module.exports = {
     'no-undef': 'error',
     'no-restricted-globals': ['error'].concat(restrictedGlobals),
     'no-unreachable': 'warn',
-    'no-unused-expressions': [
-      'error',
-      {
-        allowShortCircuit: true,
-        allowTernary: true,
-        allowTaggedTemplates: true,
-      },
-    ],
+    'no-unused-expressions': 'off',
```

Comment:
If we went for this we can remove this rule as well as the TS override.

Reply:
But is the TypeScript rule also _buggy_?

Reply:
Good point. Yeah we probably shouldn't disable for TS in that case. But since the TypeScript rule extends the eslint rule it's probably better that we get it fixed upstream as per Dan's suggestion.